tempodb/encoding/common/block.go

```diff
@@ -0,0 +1,17 @@
+package common
+
+import "strconv"
+
+const (
```

Comment:
similar thoughts here as for BlockConfig. this will all end up in v2 eventually?

Reply:
Some of it is expected to remain common across encodings, like the meta and bloom filters.  The rest, like copying the `index` file does need to move.  Wasn't in a hurry to move this yet as I think it depends on what we decide for the "entrypoint" to the backend (some options listed in the other comment)